### PR TITLE
Add Fortran string support

### LINKED ISSUE #52: Strings
Arguments of type `char *` generate the following Fortran:
```
integer(c_signed_char), dimension(*)
```

To pass c-like strings in Fortran such as `C_CHAR_"Hello World"//C_NULL_CHAR` i had to change the type to:
```
character(kind=c_char), dimension(*)
```

### PR DESCRIPTION
**Pull request by anstaf**
Fixes #52

### CODE REVIEW

File: tests/unit_tests/test_export.cpp [hunk 1]
```diff
@@ -90,6 +89,7 @@ namespace {
 
 #include <cpp_bindgen/array_descriptor.h>
 #include <cpp_bindgen/handle.h>
+#include <cpp_bindgen/string_descriptor.h>
 
 #ifdef __cplusplus
 extern "C" {
```
Comment (havogt) [inline tests/unit_tests/test_export.cpp:99]: Should we add a string_descriptor case here for completeness?

**Review by havogt — APPROVED** [review]

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch gridtools

**Comment by havogt** [comment]
launch jenkins

**Comment by havogt** [comment]
launch gridtools

File: tests/unit_tests/test_export.cpp (hunk 1 above)
Comment (anstaf) [reply tests/unit_tests/test_export.cpp:99]: done

**Comment by anstaf** [comment]
launch jenkins

**Comment by anstaf** [comment]
launch gridtools

**Review by havogt — APPROVED** [review]

**Pull request merged**